generation/internal/go.go

```diff
@@ -19,6 +34,11 @@ func goInitialSteps(ls labels.LabelSet) []config.Step {
 			Type:    config.Run,
 			Name:    "Download Go modules",
 			Command: "go mod download",
+		}, {
+			Type:    config.Run,
+			Name:    "Print go mod help instructions",
+			Command: privateModInstructions,
+			When:    config.WhenTypeOnFail,
```

Comment:
Originally I was trying detect the exit status of `go mod download` and conditionally print those instructions in the same step ( mutli line bash syntax). However I was running into errors with how go-yaml is treating multi line string (I could not figure out a way to get the library to use the `|` syntax). We ended adding a conditional step instead.